kernel/comps/systree/src/node.rs

```diff
@@ -129,7 +129,18 @@ pub trait SysBranchNode: SysNode {
 /// This trait abstracts the common interface of "normal" nodes.
 /// In particular, every "normal" node may have associated attributes.
 pub trait SysNode: SysObj {
+    /// Returns whether an attribute with the given name is absent.
+    ///
+    /// If the node does not have the attribute with the given name,
+    /// also returns `false`.
+    fn is_attr_absent(&self, _name: &str) -> bool;
```

Comment:
I think this method should live below `node_attrs`?

If you decide to change this, please change the `impl`s' method order as well.